pkgs/applications/misc/waybar/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub, meson, pkgconfig, ninja
+, wayland, wlroots, gtkmm3, libinput, libsigcxx, jsoncpp, fmt
+, traySupport  ? true,  libdbusmenu-gtk3
+, pulseSupport ? false, libpulseaudio
+, nlSupport    ? true,  libnl
+, swaySupport  ? false, sway-beta
```

Comment:
What's the closure size impact of just defaulting all of these to true?

Reply:
According to ```nix path-info -S ...```, this would change the size from 287,863,928 to 294,567,016.

Reply:
That's not a big difference. Do you think we should just enable all of them?